### PR TITLE
Feat/feedback page according to figma

### PR DESCRIPTION
**Pull request by isha-dubey**
## Description

Changed Feedback page according to figma

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by punitkashyup** [comment]
Please attach the issue link

**Comment by punitkashyup** [comment]
Please attach the issue link

File: src/packages/@deprecate/components/collections/collections-list/collection/Collection.old.svelte [hunk 1]
```diff
@@ -294,7 +294,7 @@
     showMenu = false;
   };
 
-  const onRenameInputKeyPress = (event) => {
+  const onRenameInputKeyPress = (event) => {han
```
Comment (Astitva877) [inline src/packages/@deprecate/components/collections/collections-list/collection/Collection.old.svelte:297]: what is this 'han' here?

File: src/packages/@environments/common/components/tabular-input/TabularInput.svelte [hunk 2]
```diff
@@ -27,7 +27,7 @@
     if (keyValue) {
       pairs = [];
       pairs = keyValue;
-
+      let controller: boolean = false;
```
Comment (Astitva877) [inline src/packages/@environments/common/components/tabular-input/TabularInput.svelte:30]: why we have again declared this controller here, I think it is already declared above

**Comment by punitkashyup** [comment]
Please attach the issue link

File: src/packages/@environments/common/components/tabular-input/TabularInput.svelte (hunk 2 above)
Comment (isha-dubey) [reply src/packages/@environments/common/components/tabular-input/TabularInput.svelte:30]: This is because it was needed to redeclare for the bug fix

**Comment by punitkashyup** [comment]
Please attach the issue link

**Review by itsmdasifraza — APPROVED** [review]

**Pull request merged**